### ISSUE TITLE
Offensive and demeaning

### ISSUE THREAD

**Issue by tjpalmer** [bachkukkik/awesome-thai-girls#14]
Like men, women are people, not objects.

**Comment by muyufan** [comment]
Does this page offensive to you too?
https://en.wikipedia.org/wiki/Lists_of_people

**Comment by Thrilleratplay** [comment]
From [github.com/about](https://github.com/about):
> "GitHub is how people build software.  We’re supporting a community where more than 23 million people learn, share, and work together to build software."

@muyufan Please explain how a "curated list of cute girls on Instagram or Facebook in Thailand" provides any benefit to the software development community as a whole or even small subsets of it.

While I personally find this in poor taste, I am vehemently against censorship; I will only say that Github is the wrong venue for this material.

**Comment by muyufan** [comment]
well, i agree, a list like this one does not fit on github.

but i insist that is just a classification, cute girl ugly boy White shirt black cat.

Maybe github or the author should remove the project

**Comment by t94j0** [comment]
I completely disagree that this is offensive/demeaning, but the project really doesn't belong on GitHub...

**Comment by dtinth** [comment]
Although I am not interested in these stuff, to provide some cultural context for foreigners, here’s [an article about the net idol phenomenon in Thailand](http://www.chiangmaicitylife.com/citylife-articles/a-net-full-of-idols/).

**Comment by titipata** [comment]
Hi all, thanks for all the comments. I do take all the comments seriously. 

First of all, to clarify, we didn't try to create this in offensive purpose in the beginning. However, things are getting out of control at this point. I consider closing the repository if it doesn't belong to GitHub (I will have to ask the owner of the repo about that). Really sorry again if the content is inappropriate or offensive to you all.

**Comment by bluenex** [comment]
Hi,

Thanks you guys for the honest and straightforward comments.

As part of the contributors I would like to answer the questions and also give my opinion.

> Like men, women are people, not objects.

We DO NOT objectify  and/or make fun and/or demean any people in this repo. We did what we, as a man, usually do that is sharing what we like to our friends. At first it was only for that, but now we admit that it went out of control. Differences in culture or tradition may be something we could not change or understand so easily, but we can confirm that this repo IS NOT trying to offend or demean anyone or even Github itself.

> @muyufan Please explain how a "curated list of cute girls on Instagram or Facebook in Thailand" provides any benefit to the software development community as a whole or even small subsets of it.

We may need to look at where you want it to be benefit in this case. This repo definitely gives us a chance to learn coding though it is not very high level and complex structure, it is like a relax problem to let us solve. If we look at the outcome it may be just a website with list of people, but behind there are codes that Github Pages compiles. Same goes to all the personal blog that a lot of people try to learn to make personal website using Github Pages feature. How those blogs server any benefit to the community if we do not count what is going on behind the scene.

The words we think that it is considerable and may need to take it seriously is "this kind of repo doesn't deserve to be on Github". 

Thank you.

**Comment by dtinth** [comment]
About whether it should be on GitHub or not: GitHub [showcases several open data repos](https://github.com/showcases/open-data). Therefore, this repo being unrelated to software development is probably not enough of a reason.

However, depending on [culture](https://en.wikipedia.org/wiki/Cultural_relativism), viewpoints and values, the content may or may not be perceived as [“obscene” or “discriminatory or abusive” (against GitHub ToS)](https://help.github.com/articles/github-terms-of-service/#2-content-restrictions). Stating the intentions of the project clearly may help reduce misunderstanding though.
